public/app/features/alerting/unified/components/rule-editor/PreviewRuleResult.tsx

```diff
@@ -15,6 +15,17 @@ type Props = {
 export function PreviewRuleResult(props: Props): React.ReactElement | null {
   const { preview } = props;
   const styles = useStyles2(getStyles);
+  const fieldConfig: FieldConfigSource = useMemo(() => {
```

Comment:
Do we need a `useMemo` if the values we use inside are static?

Reply:
Hm, maybe not.